### ISSUE TITLE
Button Sending but after button seen ,not received any response .

### ISSUE THREAD

**Issue by SonuSubhadip** [pedroslopez/whatsapp-web.js#2471]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

![Test45](https://github.com/pedroslopez/whatsapp-web.js/assets/131681222/dc01f956-328a-401e-98b1-7b717307be81)

I'm trying to explain in this snapshot. But i explained again I'm using normal WhatsApp but also try on business wp you can notice button can be send for button commends and it's working properly that's time. But when button recieve and see in my snap shot also trik blue because it's recieved but not get any response. Please fixed this problem. And also list not working list doesn't seen doesn't receive 

### Expected behavior

Expect Button was working properly 

### Steps to Reproduce the Bug or Issue

-----

### Browser Type

Other (please write in Additional Context)

### WhatsApp Account Type

Standard

### Does your WhatsApp account have multidevice enabled?

Yes, I am using Multi Device

### Environment

Windows 11 Home
Node js latest 
Whatsapp-web-version latest

**Comment by themazim** [comment]
Buttons and lists are blocked by meta and thus won't work. See this issue as well: https://github.com/pedroslopez/whatsapp-web.js/issues/2316

**Comment by alechkos** [comment]
Duplicate of #2316
